test/Raven/phpt/fatal_reported_with_async.phpt

```diff
@@ -21,7 +22,7 @@ $client->setSendCallback(function () {
 
 $client->install();
 
-register_shutdown_function(function () use ($pendingEvents, $pendingRequests) {
+register_shutdown_function(function () use (&$pendingEvents, &$pendingRequests) {
```

Comment:
You are using a reference for `&$pendingRequests` but I think this is still pointing at the list of requests when the script starts rather than actual client state?

Reply:
Nope, it's getting the actual client state at the end, since the test was failing until I applied 6eb54f2

Reply:
I just double-checked and no, the test will pass even if I break the code such that there are still pending requests.  $pendingRequests is a reference to the initial state of there being no pending requests, not the actual client's pending requests.

Reply:
Damn you're right once again :smile: 

Ok, so is 01a26d3 enough to fix it? It seems to fail now if I comment stuff inside the client...